docs/animated.md

```diff
@@ -49,6 +49,7 @@ const App = () => {
     Animated.timing(fadeAnim, {
       toValue: 1,
       duration: 5000,
+      useNativeDriver: true,
```

Comment:
`useNativeDriver` is required by the TS typings, and if not specified will lead to a runtime warning saying it is required.

Reply:
Should we default this to `false`? Is there a preference?